.github/release/full/Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM debian:bookworm-slim
+FROM python:3.11-slim-bullseye
```

Comment:
Debian bookworm was chosen on purpose, in order to get more modern versions of the baseline packages. Did it cause any troubles?